README.md

```diff
@@ -1 +1,231 @@
-# budget-app
\ No newline at end of file
+<div align="center">
+  <h3><b>Budget App</b></h3>
+</div>
+
+
+![mobile](https://user-images.githubusercontent.com/105460631/220864252-c1381fdf-35cd-4760-b6dd-8262b36e04e6.png)
```

Comment:
- [ ] Your images are not displaying in the README which makes them look unprofessional. Please fix this.
![image](https://user-images.githubusercontent.com/45460192/220891560-3bc1af5b-c62d-4b49-85a6-23d435bc9f4c.png)